prac_09/unreliable_car_test.py

```diff
@@ -0,0 +1,11 @@
+# Adrianto Hartono
+# 14347930
+
+from prac_09.unreliable_car import UnreliableCar
+
+def main():
+    my_unreliable = UnreliableCar("Prius 1", 100, 60)
+    my_unreliable.drive(40)
+    print(my_unreliable)
+
```

Comment:
I think this code is not completed

Reply:
Thank you for the feedback, but I think that is all🙏